### PR TITLE
Version 1 of readme updates

### PR DESCRIPTION
**Pull request by shetzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,10 +35,34 @@ To achieve a Continuous Integration and Continuous Delivery flow you can create
...
+
+To have the test runner perform setup and cleanup tasks requires the SALESFORCE_HUB_URL config var to be defined in the
+pipeline config.  Test runner uses this variable and the Salesforce buildpack generated `~/.appcloud/hubOrg.json` to
+create scratch orgs.  Within the scripts section of app.json or app-ci.json define a test script that would execute
+the test runner command from the Salesforce DX CLI, e.g.,
+```
+heroku force:test -c test/test-runner-config.json -r tap
+```
+The `test-runner-config.json` defines the setup and tear down tasks as well as the tests to be executed as part of that
+test profile.  See the [test runner schema](https://git.soma.salesforce.com/ALMSourceDrivenDev/force-com-toolbelt/blob/developer/schemas/testRunnerConfigSchema.json) for more details.
```
Comment (cwallsfdc) [inline README.md]: Let's leave out link until toolbelt is moved to public repo.

File: README.md [hunk 2]
```diff
@@ -35,10 +35,34 @@ To achieve a Continuous Integration and Continuous Delivery flow you can create
...
+
+To have the test runner simply run tests requires the Salesforce addon to be provisioned and all setup tasks to be performed before the
+tests execute.  One way to accomplish this is by defining a test-setup script within app.json or app-ci.json which handles
```
Comment (cwallsfdc) [inline README.md]: Should test-setup, app.json, app-ci.json be code formatted?

Comment (shetzel) [reply README.md]: Good call.  I'll update.

File: README.md (hunk 1 above)
Comment (shetzel) [reply README.md]: I'll remove.

**Pull request merged**